libs/blocks/marketo/marketo.js

```diff
@@ -15,6 +15,7 @@
  */
 import { parseEncodedConfig, loadScript, createTag, createIntersectionObserver } from '../../utils/utils.js';
 
+const ROOT_MARGIN = 1000;
```

Comment:
Any insight on how this was chosen?

Reply:
Since the footer is the last element on the page, with low interaction, and we load it after a set timeout anyway, we had the luxury to hold off a bit more and use a lower rootMargin value. But it is a valid question, a high `rootMargin` value could potentially load things a bit eagerly. Where on the page is this being generally used? How long does it take to load it? That may also justify a higher value, to ensure the section is decorated before the user reaches that point.

Reply:
This is being used on resource pages and we will be migrating FaaS forms to Marketo forms so value this was taken from the FaaS block. Most of these forms appear above the fold or at the bottom of the page. There is some load time for the form and this value may be too high but it's a good place to start until we have more data.